quantum/nodes/hadamard-gate/hadamard-gate.html

```diff
@@ -0,0 +1,36 @@
+<script type="text/javascript">
+    RED.nodes.registerType("hadamard-gate", {
+      category: "quantum",
+      color: "#FA4D56",
+      defaults: {
+        name: { value: "H" },
```

Comment:
```suggestion
        name: { value: "" },
```
Ideally name default should be left blank.